docker_test.go

```diff
@@ -1134,6 +1164,10 @@ func ExampleContainer_Stop() {
 	}()
 	timeout := 10 * time.Second
 	_ = nginxC.Stop(ctx, &timeout)
+
+	fmt.Print("you have a running and stop the container")
+
+	// Output: you have a running and stop the container
```

Comment:
Could you apply this to https://github.com/testcontainers/testcontainers-go/pull/1825/files#diff-037b83426e676aa3786c08b334b0db7bb70235792961cbe4ed3340bc0a60a7a8R1157? 🙏  
```
	if err != nil {
		panic(err)
	}
```

I think we can improve the output like this:
```suggestion
	fmt.Println("Container has been started")
	timeout := 10 * time.Second
	err = nginxC.Stop(ctx, &timeout)
	if err != nil {
		panic(err)
	}

	fmt.Println("Container has been stopped")

	// Output:
	// Container has been started
	// Container has been stopped
```